tests/functional/test_dependencies.py

```diff
@@ -95,6 +96,34 @@ def test_dependency_using_reference(ref, recwarn, dependency_manager):
     assert DeprecationWarning not in [w.category for w in recwarn.list]
 
 
+def test_get_npm_version_from_spec():
+    assert get_npm_version_from_spec("1.0") == "==1.0.0"
+    assert get_npm_version_from_spec("1") == "==1.0.0"
+    assert get_npm_version_from_spec("1.0.0-beta") == "==1.0.0-beta"
+    assert get_npm_version_from_spec("1.0-beta") == "==1.0.0-beta"
+    assert get_npm_version_from_spec("1-beta") == "==1.0.0-beta"
+    assert get_npm_version_from_spec("1.0.0-beta.1") == "==1.0.0-beta.1"
+    assert get_npm_version_from_spec("1.0-beta.1") == "==1.0.0-beta.1"
+    assert get_npm_version_from_spec("1-beta.1") == "==1.0.0-beta.1"
+
+    assert get_npm_version_from_spec("~=1.0") == "~=1.0"
+    assert get_npm_version_from_spec(">=1") == ">=1.0"
+    assert get_npm_version_from_spec("<=1.0.0-beta") == "<=1.0.0-beta"
+    assert get_npm_version_from_spec(">1.0-beta") == ">1.0.0-beta"
+    assert get_npm_version_from_spec("<1-beta") == "<1.0.0-beta"
+
+
+def test_npm_version_from_spec():
+    name = "@gnosis.pm"
+    package = "safe-singleton-factory"
+    version = "==1.0.0"
+    dependency = NpmDependency(name=package, npm=f"{name}/{package}", version=version)
+    assert dependency.version == get_npm_version_from_spec("1.0.0")
+    assert dependency.version == get_npm_version_from_spec(" 1.0.0")
+    assert dependency.version == get_npm_version_from_spec(" == 1.0.0")
```

Comment:
it's a little odd to be testing both sides of the assert,
typically left side represents the actual value and the right side is like a hardcoded expectation.

i think the right-side checks here are handled in the test above anyway!